packages/teams-js/src/public/runtime.ts

```diff
@@ -215,6 +215,76 @@ interface IRuntimeV3 extends IBaseRuntime {
   };
 }
 
+interface IRuntimeV4 extends IBaseRuntime {
```

Comment:
>IRuntimeV4 [](http://example.com/codeflow?start=10&length=10)

FYI depending on who checks in first, PR #2023 may also need to be updated since it makes runtime changes.